runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridConfig.java

```diff
@@ -24,17 +25,34 @@
  */
 package net.runelite.client.plugins.inventorygrid;
 
+import net.runelite.client.config.Alpha;
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
+import java.awt.*;
+
 @ConfigGroup("inventorygrid")
 public interface InventoryGridConfig extends Config
 {
+
+
+	@ConfigItem(
+		keyName = "showindividualGrid",
+		name = "Toggle individual grid",
+		description = "Shows boxes where items go",
+		position = 5
```

Comment:
Whats up with all of the config positions being backwards?

Reply:
They was rendering backwards for some reason when I did it the other way

Reply:
Items with a lower priority are higher on the list, and we traditionally keep the config items ordered in source the same way their priorities are.